postinstall/Makefile

```diff
@@ -2,6 +2,8 @@ TOP=..
 include $(TOP)/configure/CONFIG
 
 install:
+	mkdir -p /usr/local/share/misc
+	ln -s $(shell realpath $(TOP))/iocBoot/iocCamera/envPaths /usr/local/share/misc/
```

Comment:
We could accept a `PREFIX` argument here, I think. Just a tad cleaner :)

Could you also add a comment to the commit message that `envPaths` is arch-independent because it only lists the root directory for each module, not any internal build directory?

Reply:
> We could accept a `PREFIX` argument here, I think. Just a tad cleaner :)

Seems nice. Would `PREFIX ?= /usr/local` do a good job here or is there a better way to parameterize this?

> Could you also add a comment to the commit message that `envPaths` is arch-independent because it only lists the root directory for each module, not any internal build directory?

Sure. Will do.

Reply:
> **PREFIX ?= /usr/local**

Good enough. If you want to go further you can define `bindir` and `sharedir` based on PREFIX so even those specific directories can be overridden, but I think that's unnecessary.